main.java

```diff
@@ -0,0 +1,149 @@
+package hangman;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Random;
+import java.util.Scanner;
+
+public class Main {
+	
+	public static String art(int i) {
+		String[] hangmanStages = {
+				"""
+				 =========
+				""","""
+		           +---+
+		           |   |
+		               |
+		               |
+		               |
+		               |
+		         =========
+		         """, """
+		           +---+
+		           |   |
+		           O   |
+		               |
+		               |
+		               |
+		         =========
+		         """, """
+		           +---+
+		           |   |
+		           O   |
+		           |   |
+		               |
+		               |
+		         =========
+		         """, """
+		           +---+
+		           |   |
+		           O   |
+		          /|   |
+		               |
+		               |
+		         =========
+		         """, """
+		          +---+
+		           |   |
+		           O   |
+		          /|\\  |
+		               |
+		               |
+		         =========		
+		         """, """
+				  +---+
+				  |    |
+				  O    |
+				 /|\\   |
+				 /     |
+				       |
+				=========
+				""", """
+				  +---+
+				  |    |
+				  O    |
+				 /|\\   |
+				 / \\   |
+				       |
+				=========
+		        """};
+		return hangmanStages[i];
+		}
+
+	public static String randomWord() {
+		
+		String[] words = {"useful", "random", "hangman", "console", "application", "represent", "underscore", "example", "reveal"};
+		
+		Random random = new Random();
+		int randIndex = random.nextInt(words.length);
+		
+		return words[randIndex];
+	}
+	
+	
+	public static void main(String[] args) {
+		
+	 // variables setup
+		
+	 String rightAnswer = randomWord();
+	 Integer amountOfGuesses = 7;
+	 Integer amountOfWrongGuesses = 0;
+	 
+	 ArrayList<String> splitAnswer = new ArrayList<String>(Arrays.asList(rightAnswer.split("")));
+	 ArrayList<String> hiddenAnswer = new ArrayList<String>();
+	 
+	 splitAnswer.forEach(letter -> {hiddenAnswer.add("_");});
+	 
+	 // game logic
+
+	 
+	 while(amountOfWrongGuesses < amountOfGuesses && hiddenAnswer.contains("_")){
+		 System.out.println(art(amountOfWrongGuesses));
+		 System.out.println(hiddenAnswer);
+		 
+		 Scanner s = new Scanner(System.in);
```

Comment:
Create a class that is only concerned with taking input. 
has methods like 'takeValidInput'

Reply:
You could hold guesses that have already happened in an array